Upload A File/Download Upload Library.gcomp/DownloadOrUploadFromWebVI.jsli

```diff
@@ -46,113 +46,12 @@
 					<JavaScriptLibraryDiagramParameter CallDirection="Input" Id="fffab26d34a14570ac6c8d576f379b72" IsError="True" Name="error in" />
 					<JavaScriptLibraryDiagramParameter CallDirection="Output" Id="feb40accc9f54a5fb7cf51a2136b764d" IsError="True" Name="error out" />
 					<JavaScriptLibraryParameter Id="309f6339715946c5af965d3ec4fde639" IsReturnParameter="True" Name="return">
-						<JavaScriptType ActualDataType="@8c004d347e8d44d38bf49e479e68c74c" Id="21463e0ff0a549308e18b32c54b7ed36" />
```

Comment:
The main change here is that a lot of the functions needed for polling are no longer needed and removed from the JSLI

![image](https://user-images.githubusercontent.com/1588923/61586204-ea0e5380-ab33-11e9-9c0e-b064b00ccea0.png)